language-configuration-erb.json

```diff
@@ -0,0 +1,28 @@
+{
+	"comments": {
+		"blockComment": [ "<%#", "%>" ]
```

Comment:
No, you're probably right. Perhaps leave the HTML comment and let the ruby language processing pickup the comment inside the erb tags.